src/utils.ts

```diff
@@ -11,3 +11,7 @@ export function decimalKeypad(numericValue: string): string {
   const number = numericValue.replace(',', '.');
   return number.split('.').length <= 2 ? number : number.slice(0, -1);
 }
+
+export function countUtf8Bytes(str: string): number {
```

Comment:
Can you add a test for this? Or create an issue to add it

Reply:
https://github.com/litentry/litentry-app/issues/1186